src/Wallabag/CoreBundle/Command/InstallCommand.php

```diff
@@ -254,7 +254,7 @@ protected function setupAdmin()
         $question->setHidden(true);
         $user->setPlainPassword($this->io->askQuestion($question));
 
-        $user->setEmail($this->io->ask('Email', ''));
+        $user->setEmail($this->io->ask('Email', 'wallabag@wallabag.io'));
```

Comment:
Why is this here?

Reply:
Sorry, didn't explain.
It's because of this: https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php#L46

We can't define an ask question without a default value which isn't empty (as we used before).

Reply:
Okay, but still odd for this change to be in the PR.

Reply:
I agree but it was broken the tests.